mac.go

```diff
@@ -51,7 +51,8 @@ func verifyMac(macData *macData, message, password []byte) error {
 	return nil
 }
 
-// Error indicating that the supplied password is incorrect. Usually, P12/PFX data is signed to be able to verify the password.
+// PasswordIncorrect Error indicates that the supplied password is incorrect.
+// Usually, P12/PFX data is signed to be able to verify the password.
 type PasswordIncorrect string
```

Comment:
Usually these types have an Error suffix (PasswordIncorrectError), but I avoided changing your API and breaking the callers.

Alternatively, if the error doesn't need to be dynamic, you could just export an error variable instead of creating a type:
```go
var (
    ErrPasswordIncorrect = errors.New("Incorrect password, MAC mismatch")
)
```